lib/msal-common/src/network/NetworkManager.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ServerAuthorizationTokenResponse } from "../server/ServerAuthorizationTokenResponse";
+import { INetworkModule } from "./INetworkModule";
+import { ServerTokenRequestParameters } from "../server/ServerTokenRequestParameters";
+import { ICacheStorage } from "../cache/ICacheStorage";
+import { AccessTokenKey } from "../cache/AccessTokenKey";
+import { RequestThumbprint } from "./RequestThumbprint";
+import { Constants } from "../utils/Constants";
+import { ICrypto } from "../crypto/ICrypto";
+import { TokenExchangeParameters } from "../request/TokenExchangeParameters";
+import { ClientInfo } from "../auth/ClientInfo";
+import { ErrorValue } from "../cache/ErrorValue";
+
+export type NetworkResponse = {
+    headers: Headers;
+    body: string;
+    status: number;
+};
+
+export class NetworkManager {
+    cacheStorage: ICacheStorage;
```

Comment:
would it be better to instead pass in a `CacheHelpers` and implement the related functions in that class, rather than directly writing to the cache `cacheStorage` here?